public/assets/js/client.js

```diff
@@ -1,16 +1,16 @@
 //make the post request to '/register'
 function signUpSubmit(e) {
     e.preventDefault()
-    let firstName = document.getElementById('first_name').value.trim()
-    let lastName = document.getElementById('last_name').value.trim()
-    let mobile = document.getElementById('mob_no').value.trim()
-    let userName = document.getElementById('user_name').value.trim()
-    let password = document.getElementById('password').value.trim()
-    let confirmPassword = document.getElementById('confirm-password').value.trim()
-    let address = document.getElementById('address').value.trim()
-    let zipcode = document.getElementById('zipcode').value.trim()
+    const firstName = document.getElementById('first_name').value.trim()
+    const lastName = document.getElementById('last_name').value.trim()
+    const mobile = document.getElementById('mob_no').value.trim()
+    const userName = document.getElementById('user_name').value.trim()
+    const password = document.getElementById('password').value.trim()
+    const confirmPassword = document.getElementById('confirm-password').value.trim()
+    const address = document.getElementById('address').value.trim()
+    const zipcode = document.getElementById('zipcode').value.trim()
```

Comment:
`const` by default, not `let`.  These values never change, so `let` isn't needed here.